duckduckgo/iOS
# (Hopefully) Fix tracker reports sometimes being incorrect due to race condition

Task/Issue URL: https://app.asana.com/0/414709148257752/1200297306793824/f
Tech Design URL:
CC:

**Description**:
A potential fix, but I haven't been able to recreate the original issue to know for sure, so will have to wait for Bartek's return

**Steps to test this PR**:
1. Disable the dax logo on the SERP so you can see the site rating
1. Attempt to make it so a tracker is (incorrectly) reported on the SERP. If this works you shouldn't succeed.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
